swagger/public_v1/api_spec.json

```diff
@@ -2002,4 +1988,4 @@
       }
     }
   }
-}
+}
```

Comment:
line ending

Reply:
Seems ok - at some point prior some non-relevant attrs were removed

<img width="994" alt="docs" src="https://user-images.githubusercontent.com/5256922/104590964-f493a580-5663-11eb-8a99-f21764c43fcc.png">


This change also appears  in @darokel 's PR
https://github.com/DFE-Digital/teacher-training-api/pull/1716/files#diff-9837f120202fbbaabd948d41e32926fffab43b8fe3b270840bf901aa1cc69321L1065-L1081